index.js

```diff
@@ -1,6 +1,30 @@
 
 import { NativeModules } from 'react-native';
+import { Buffer } from 'buffer';
 
 const RNUportSigner = NativeModules.RNUportSignerModule;
 const RNUportHDSigner = NativeModules.RNUportHDSignerModule;
-export { RNUportSigner, RNUportHDSigner };
+const DerivationPathRoot = RNUportHDSigner.UPORT_ROOT_DERIVATION_PATH;
+
+function getSignerForAddress(address, derivationPath = DerivationPathRoot, userPrompt = "") {
+  return async (data) => {
+    const { v, r, s } = await RNUportHDSigner.signJwt(address,
+      derivationPath,
+      Buffer.from(data).toString('base64'),
+      'Sign a JWT');
+    
+    return {
+      recoveryParam: ((v == 0 || v == 1) ? v : v - 27),
+      r: Buffer.from(r, 'base64').toString('hex'),
+      s: Buffer.from(s, 'base64').toString('hex')
+    }
+  }
+}
+
+function constructPath(hdIndex = 0, acctIndex = 0, recovery = 0) {
```

Comment:
Not sure we want to expose this method as it is internal to the uPort app account logic.